test/graphql/types/Mutation/updateAgendaItem.test.ts

```
@@ -0,0 +1,299 @@
+		agendaItemsTable: {
+			findFirst: vi.fn(),
+		},
+		agendaFoldersTable: {
+			findFirst: vi.fn(),
+		},
+	},
+	update: vi.fn().mockReturnThis(),
+	set: vi.fn().mockReturnThis(),
+	where: vi.fn().mockReturnThis(),
+	returning: vi.fn(),
+} as any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)